src/main/java/io/confluent/examples/streams/GlobalStoresExample.java

```diff
@@ -0,0 +1,242 @@
+import io.confluent.examples.streams.avro.Order;
+import io.confluent.examples.streams.avro.Product;
+import io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.kstream.Consumed;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.Produced;
+import org.apache.kafka.streams.kstream.ValueTransformer;
+import org.apache.kafka.streams.processor.Processor;
+import org.apache.kafka.streams.processor.ProcessorContext;
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.apache.kafka.streams.state.Stores;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Demonstrates how to perform "joins" between  KStreams and GlobalStore, i.e. joins that
+ * don't require re-partitioning of the input streams.
+ * <p>
+ * In this example, we join a stream of orders that reads from a topic named
+ * "order" with a customers global store that reads from a topic named "customer", and a products
+ * global store that reads from a topic "product". The join produces an EnrichedOrder object.
+ * <p>
+ * <br>
+ * HOW TO RUN THIS EXAMPLE
+ * <p>
+ * 1) Start Zookeeper, Kafka, and Confluent Schema Registry. Please refer to <a href='http://docs.confluent.io/current/quickstart.html#quickstart'>QuickStart</a>.
+ * <p>
+ * 2) Create the input/intermediate/output topics used by this example.
+ * <pre>
+ * {@code
+ * $ bin/kafka-topics --create --topic order \
+ *                    --zookeeper localhost:2181 --partitions 4 --replication-factor 1
+ * $ bin/kafka-topics --create --topic customer \
+ *                    --zookeeper localhost:2181 --partitions 3 --replication-factor 1
+ * $ bin/kafka-topics --create --topic product \
+ *                    --zookeeper localhost:2181 --partitions 2 --replication-factor 1
+ * $ bin/kafka-topics --create --topic enriched-order \
+ *                    --zookeeper localhost:2181 --partitions 4 --replication-factor 1
+ * }</pre>
+ * Note: The above commands are for the Confluent Platform. For Apache Kafka it should be
+ * `bin/kafka-topics.sh ...`.
+ * <p>
+ * 3) Start this example application either in your IDE or on the command line.
+ * <p>
+ * If via the command line please refer to <a href='https://github.com/confluentinc/kafka-streams-examples#packaging-and-running'>Packaging</a>.
+ * Once packaged you can then run:
+ * <pre>
+ * {@code
+ * $ java -cp target/kafka-streams-examples-6.0.1-standalone.jar io.confluent.examples.streams.GlobalStoresExample
+ * }
+ * </pre>
+ * 4) Write some input data to the source topics (e.g. via {@link GlobalKTablesExampleDriver}). The
```

Comment:
Should we point out that we reuse this diver for two examples? Otherwise, the name might be confusing? Or rename the driver class?

Reply:
Updated the doc and renamed the class `GlobalKTablesAndStoresExampleDriver`.